repo/migrations/Migration008_test.go

```diff
@@ -122,8 +122,8 @@ func TestMigration008(t *testing.T) {
 		saleID, // sale order id
 		"",     // sale contract blob
 		migrations.Migration008_OrderState_AWAITING_PAYMENT, // order state int
-		0,                      // purchase read bool
-		0,                      // sale read bool
+		0, // purchase read bool
+		0, // sale read bool
```

Comment:
Watching this pattern change back and force is really noisy in our git history as well as for review purposes. Can you find out what env inconsistency could be causing this or take more care to not commit these changes (and then reversals) to git?

Reply:
It's because we're still using gofmt 10.3. I'd like to upgrade to go 11 if we can.